hnsecure/src/test/java/ca/bc/gov/hlth/hnsecure/messagevalidation/V2PayloadValidatorTest.java

```diff
@@ -174,11 +167,10 @@ public void testPNPMessageFormat() {
         
     }
 
-
-    @Test  
+    // Validate that No exception is thrown 
+    @Test (expected = Test.None.class) 
```

Comment:
I'm fine with this approach (as I believe it was chosen since there aren't actually any assertions in the method) but it typically isn't necessary because any exceptions NOT declared as expected will make the test fail anyways.

Reply:
Updated